data/json/npcs/exodii/common_talk.json

```diff
@@ -46,15 +46,41 @@
     "type": "talk_topic",
     "id": "TALK_Directions_exodii_true",
     "//": "For exodii 'find warehouse' mission, used by NPCs giving directions.",
-    "dynamic_line": "&You listen to the directions, and jot them down in your map.",
+    "dynamic_line": {
+      "concatenate": [
+        "&You listen to the directions, and jot them down in your map",
+        {
+          "and": [ { "math": [ "rng(1, 3)", ">", "1" ] }, { "follower_present": "NC_STARTINGNPC_LIAM" } ],
```

Comment:
```suggestion
          "and": [ { "one_in_chance": 2 }, { "follower_present": "NC_STARTINGNPC_LIAM" } ],
```

Reply:
i see an error now when loading the game related to this line. i dont really understand enough about eoc to know whats wrong though